spec/index.bs

```diff
@@ -701,12 +714,24 @@ initially empty.
     1. If |elementNs| is an empty string, set |elementNs| to [=HTML namespace=].
     1. If |attrNs| is an empty string, set |attrNs| to null.
     1. Let |interface| be the [=element interface=] for |localName| and |elementNs|.
-    1. If |interface| does not have an IDL <a spec="webidl">attribute</a> that [=reflects=] a content attribute with
-        |localName| local name and |attrNs| namespace,
-        return undefined and abort further steps. Otherwise, let |idlAttribute| be that IDL <a spec="webidl">attribute</a>.
-    1. If {{StringContext}} IDL extended attribute appears on |idlAttribute|, return
-        stringified {{StringContext}}'s identifier and abort further steps.
-    1. Return null.
+    1. Let |expectedType| be null.
+    1. Find the row in the following table, where |interface|'s name is in the first column, and |attribute|'s <a for="Attr">namespace</a> and <a for="Attr">local name</a> match the values in the second and third column, respectively.
+        If a matching row is found, set |expectedType| to the value of the fourth.
+
+        <table>
+          <thead>
+            <tr><th>Element<th>Attribute namespace<th>Attribute local name<th>TrustedType
```

Comment:
This is duplicated from Get trusted type compliant attribute value, it should probably be shared somehow (the other function needs a sink name too else I'd make an alogrithm that returns the value).

Could I make a shared algorithm that returns the row somehow?